tests/Helper/ScriptsTest.php

```diff
@@ -25,12 +25,14 @@ public function test()
 
         $scripts->add('/js/last.js', 150);
         $scripts->add('/js/first.js', 50);
+        $scripts->add('/js/attr.js', 100, array('foo' => true, 'bar' => 'baz'));
```

Comment:
I think the attr escaper will do `<script foo="true"` if you pass the string value instead of a literal boolean value, no?